protos/follow_me/follow_me.proto

```diff
@@ -30,19 +30,23 @@ service FollowMeService {
 
 // Configuration type.
 message Config {
-    // Direction relative to the target that the vehicle should follow
-    enum FollowDirection {
-        FOLLOW_DIRECTION_NONE = 0; // Do not follow
-        FOLLOW_DIRECTION_BEHIND = 1; // Follow from behind
-        FOLLOW_DIRECTION_FRONT = 2; // Follow from front
-        FOLLOW_DIRECTION_FRONT_RIGHT = 3; // Follow from front right
-        FOLLOW_DIRECTION_FRONT_LEFT = 4; // Follow from front left
+    // Follow Perspective that the vehicle should follow
+    enum FollowPerspective {
+        FOLLOW_PERSPECTIVE_BEHIND = 0;      // Follow from behind
+        FOLLOW_PERSPECTIVE_FRONT = 1;       // Follow from front
+        FOLLOW_PERSPECTIVE_FRONTRIGHT = 2;  // Follow from front right
+        FOLLOW_PERSPECTIVE_FRONTLEFT = 3;   // Follow from front left
+        FOLLOW_PERSPECTIVE_RIGHT = 4;       // Follow from right
+        FOLLOW_PERSPECTIVE_LEFT = 5;        // Follow from left
+        FOLLOW_PERSPECTIVE_BEHINDRIGHT = 6; // Follow from behind right
+        FOLLOW_PERSPECTIVE_BEHINDLEFT = 7 ; // Follow from behind left
+        FOLLOW_PERSPECTIVE_INVALID = 8;     // Follow perspective invalid
     }
 
-    float min_height_m = 1 [(mavsdk.options.default_value)="8.0"]; // Minimum height for the vehicle in meters (recommended minimum 8 meters)
+    float follow_height_m = 1 [(mavsdk.options.default_value)="8.0"]; // Minimum height for the vehicle in meters (recommended minimum 8 meters)
     float follow_distance_m = 2 [(mavsdk.options.default_value)="8.0"]; // Distance from target for vehicle to follow in meters (recommended minimum 1 meter)
-    FollowDirection follow_direction = 3; // Direction to follow in
-    float responsiveness = 4 [(mavsdk.options.default_value)="0.5"]; // How responsive the vehicle is to the motion of the target (range 0.0 to 1.0)
+    FollowPerspective follow_perspective = 0; // Perspective to follow
```

Comment:
It should stay `= 3`. The number here is for protobuf: it is not the default value of the enum :-)

Reply:
I would leave the variable name but change the comment to make it clearer.